cmd/kops/validate_cluster.go

```diff
@@ -88,7 +97,9 @@ func RunValidateCluster(f *util.Factory, cmd *cobra.Command, args []string, out
 		return fmt.Errorf("cannot get InstanceGroups for %q: %v", cluster.ObjectMeta.Name, err)
 	}
 
-	fmt.Fprintf(out, "Validating cluster %v\n\n", cluster.ObjectMeta.Name)
+	if options.output == OutputTable {
```

Comment:
Since this is specific to the OutputTable, does it make sense to move it to its case section of the switch below?

Reply:
Yes!

Reply:
Well I would have to add it in two places ;(

Reply:
yep I see what you mean.